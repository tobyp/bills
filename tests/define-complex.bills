#!./bills
A	X.	1.00

%define X A(2B3C)4(D(E))

A	2X3(BCDE).	300.00
