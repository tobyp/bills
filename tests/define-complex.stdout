todo: B pays 53.0 to A
todo: C pays 57.0 to A
todo: D pays 85.0 to A
todo: E pays 85.0 to A
todo: X pays 1.0 to A
